projects/packages/jetpack-mu-wpcom/src/features/cloudflare-analytics/cloudflare-analytics.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Cloudflare Analytics
+ * Let WPCOM users automatically insert a Cloudflare analytics JS snippet into their site footer.
+ *
+ * @since $$next-version$$ -- Ported from Jetpack.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom\Cloudflare_Analytics;
+
+/**
+ * Add Cloudflare Analytics tracking code to the head.
+ *
+ * @since $$next-version$$ -- Ported from Jetpack.
+ */
+function insert_tracking_id() {
+	$option = get_option( 'jetpack_cloudflare_analytics' );
+
+	if (
+		! empty( $option['code'] )
+		&& ! is_admin()
+		&& ( class_exists( 'Jetpack_AMP_Support' ) && ! \Jetpack_AMP_Support::is_amp_request() )
```

Comment:
Not sure if `Jetpack_AMP_Support` should always exist, but if it doesn't this will return `false`. Should we maybe update this to: ``
```suggestion
		&&  ( ! class_exists( 'Jetpack_AMP_Support' ) || ( class_exists( 'Jetpack_AMP_Support' ) && ! \Jetpack_AMP_Support::is_amp_request() ) )
```

Reply:
I would expect it to be always available today, but you're right that it doesn't hurt to be extra defensive, it's more future-proof.